config.py

```diff
@@ -91,7 +91,7 @@ class Development(Config):
     DM_API_AUTH_TOKENS = 'myToken'
     DM_API_CALLBACK_AUTH_TOKENS = 'myToken'
     DM_SEARCH_API_AUTH_TOKEN = 'myToken'
-    DM_SEARCH_API_URL = 'http://localhost:5001'
+    DM_SEARCH_API_URL = f"http://localhost:{os.getenv('DM_SEARCH_API_PORT', 5001)}"
```

Comment:
I think we need to pick a default port and stick with it across the repos/manual etc - 5001 or 5009.

Reply:
I think we should change the default to 5009, although not necessarily using this method (depending on what the team thinks).

@benvand has already made a PR for the manual https://github.com/alphagov/digitalmarketplace-manual/pull/273

Reply:
It's more about removing 5001 across our repos to avoid confusion (and it matching what's in the manual - good work @benvand!)

Reply:
I've spotted one other repo where we use port 5001 - the router's `make test-nginx` command. It's just passing in dummy env variables to the docker image before building it so not a blocker, but if we're removing 5001 everywhere that's another one for the list 😸

Reply:
I hadn't done all the repos thoroughly as was waiting for reviews, I will create a PR for the above when I get a moment though